packages/contracts/deploy/00_managing-dao/20_set-dao-permission.ts

```diff
@@ -0,0 +1,33 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DeployFunction} from 'hardhat-deploy/types';
+
+import {DAO_PERMISSION, getContractAddress, managePermission} from '../helpers';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const {getNamedAccounts, ethers} = hre;
+  const {deployer} = await getNamedAccounts();
+
+  // Get `managingDAO` address.
+  const managingDAOAddress = await getContractAddress('DAO', hre);
+
+  // Get `DAO` contract.
+  const managingDaoContract = await ethers.getContractAt(
+    'DAO',
+    managingDAOAddress
+  );
+
+  // Set all the permission needed for a DAO to operate normally as if it was created via DAOFactory.
+  for (let index = 0; index < DAO_PERMISSION.length; index++) {
+    const permission = DAO_PERMISSION[index];
```

Comment:
Well, this will take huge time on `mainnet` as we await for each tx. but good news. 

do the `applySingleTargetPermissions` and the problem is solved.

Reply:
amended